manifests/minimal-postgres-manifest.yaml

```diff
@@ -15,5 +15,7 @@ spec:
     foo_user: []  # role for application foo
   databases:
     foo: zalando  # dbname: owner
+  preparedDatabases:
+    bar: {}
```

Comment:
does this have the effect of creating the bar DB owned by `db_owner` ?

Reply:
The roles are always called like the DB. So it would be `bar_owner` etc.
Plus, there will be a schema called `data`